ingestion-beam/src/main/java/com/mozilla/telemetry/republisher/RepublisherOptions.java

```diff
@@ -143,7 +144,8 @@ static void enrichRepublisherOptions(Parsed options) {
     options
         .setDeduplicateExpireSeconds(NestedValueProvider.of(options.getDeduplicateExpireDuration(),
             value -> Ints.checkedCast(Time.parseSeconds(value))));
-    options.setParsedRedisUri(NestedValueProvider.of(options.getRedisUri(), URI::create));
+    options.setParsedRedisUri(NestedValueProvider.of(options.getRedisUri(),
+        s -> Optional.ofNullable(s).map(URI::create).orElse(null)));
```

Comment:
This now matches exactly what we do in `DecoderOptions`. I just forgot that RepublisherOptions needs to do the same thing.